microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md

```diff
@@ -0,0 +1,158 @@
+---
+title: What's new in DevTools (Microsoft Edge 103)
+description: "New node types in the Memory tool. DevTools extension for Visual Studio Code: Right-click an HTML file to launch, and Application tool. Focus Mode bug fixes. 3D View tool responsive layout. Fix: The Close button for DevTools is now accessible in high contrast mode. Welcome tool redesign. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 06/21/2022
+---
+# What's New in DevTools (Microsoft Edge 103)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+> [!TIP]
+> The **Microsoft Build 2022** conference was on May 24-26.  Learn more about new features for memory, performance, and production debugging in DevTools and new capabilities in the web platform for PWAs in the following videos:
+> * [Tips for debugging memory, performance, & production issues](https://www.youtube.com/watch?v=hdrR0QwXpuc)
+> * [Microsoft Edge | Build 2022: Enriching desktop experiences with the power and reach of the web](https://www.youtube.com/watch?v=ahO5nePl4BQ)
+
+
+<!-- ====================================================================== -->
+## New node types in the Memory tool
+
+<!-- Title: New "object shape" and "BigInt" node types in the Memory tool -->
+<!-- Subtitle: When taking a heap snapshot, you can now filter to new node types representing hidden classes and descriptor arrays in V8, the JavaScript engine of Microsoft Edge. -->
+
+In Microsoft Edge 103, when taking a heap snapshot in the **Memory** tool, there are two new node types:
+*  **Object shape** - Refers to the hidden classes and descriptor arrays that V8 (the JavaScript engine of Microsoft Edge) uses to understand and index the properties on objects. 
+*  **BigInt** - Used to represent and manipulate values that are too large to be represented by the **Number** node type.
+
+![BigInt and Object Shape memory node types.](devtools-103-images/memory-node-types.png)
+
+For history, see [Issue 12769: Improve categorization of objects in heap snapshots](https://bugs.chromium.org/p/v8/issues/detail?id=12769).
+
+See also:
+* [Constructor (group) entries in the heap profiler](../../../memory-problems/heap-snapshots.md#constructor-group-entries-in-the-heap-profiler) in _Record heap snapshots using the Memory tool_.
+* [HiddenClasses and DescriptorArrays](https://v8.dev/blog/fast-properties#hiddenclasses-and-descriptorarrays) - Fast properties in V8.
+* [BigInt object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt).
+
+
+<!-- ====================================================================== -->
+## DevTools extension for Visual Studio Code: Right-click an HTML file to launch, and Application tool
```

Comment:
I think we can bury the lead a little bit here about the VS Code extension in favor of a shorter, clearer title. The title still has the value prop of being able to do these things directly from VS Code

```suggestion
## Launch the browser from an HTML file and use the Application tool within Visual Studio Code
```